src/hotspot/share/libadt/vectset.hpp

```diff
@@ -55,6 +55,9 @@ class VectorSet : public AnyObj {
   VectorSet(Arena* arena);
   ~VectorSet() {}
 
+  // Allow move constructor for && (eg. capture return of function)
+  VectorSet(VectorSet&&) = default;
```

Comment:
Same as the other, consider invalidating the moved from `VectorSet` by setting the data to nullptr or something similar to catch misbehaving code.